timeseries/tests/smoketests/test_all_models.py

```diff
@@ -157,7 +156,7 @@ def test_all_models_can_handle_all_covariates(
         use_known_covariates=use_known_covariates,
         use_past_covariates=use_past_covariates,
         use_static_features_continuous=use_static_features_continuous,
-        use_static_features_categorical=use_static_features_categorical,
+        use_static_features_categorical=False,
```

Comment:
I would prefer to set `use_static_features_continuous=False` and keep `use_static_features_categorical` as a test case so that we test at least some categorical features.